.github/workflows/ci.yaml

```diff
@@ -4,6 +4,8 @@ on:
   pull_request:
   push:
     branches: [ main ]
+    # don't run when pushing tags
+    tags-ignore: [ "*" ]
```

Comment:
ci.yaml doesn't need to run when pushing the tag because it has already run as part of a PR or when changes are merged directly to master